app/assets/stylesheets/components/_language_selector.scss

```diff
@@ -0,0 +1,16 @@
+.language-selector__select {
+  margin-bottom: $baseline-unit*2;
+}
+
+.language-selector__delete {
+  margin-left: 0.5em;
+  cursor: pointer;
```

Comment:
I'd tend to agree. The only fly in the ointment is that it's within a form, and buttons in a form, by default, submit that form — so we'll have to interfere with that behaviour. 

Before I've also encountered the first button in a form receiving a click event when the form is submitted by hitting enter on a text field elsewhere — but that was in pensions calculator so could have been for many other reasons...

Might give it a go anyway